graphs/graph_list.py

```diff
@@ -1,10 +1,24 @@
-#!/usr/bin/env python3
```

Comment:
Why remove this?

Instead of writing over Omkar and Nwachukwu's work, can you please put this in a separate file with a different name and leave their work intact?

Reply:
Sorry, didn't mean to delete that line. I can definitely leave their work intact, but the original issue was written to improve on their work, so I figured it was okay to edit and improve it. I can make separate file though!

Reply:
Wait, do you want me to make a separate file for the graph_matrix.py too? Because I did overwrite pretty much all the work in that old file (but there was no author written there) @cclauss

Reply:
I am OK to whack graphs/graph_matrix.py